src/OrchardCore.Modules/OrchardCore.Menu/Migrations/menu.recipe.json

```diff
@@ -98,6 +132,34 @@
               "Description": "Provides a menu link behavior for your content item."
             }
           }
+        },
+        {
+          "Name": "ContentPickerMenuItemPart",
+          "Settings": {
+            "ContentPartSettings": {
+              "Attachable": true,
+              "DisplayName": "ContentPickerMenuItem",
+              "Description": "Provides a menu link behavior from ContentPicker for your content item."
+            }
+          },
+          "ContentPartFieldDefinitionRecords": [
+            {
+              "FieldName": "ContentPickerField",
+              "Name": "LinkToContentItem",
+              "Settings": {
+                "ContentPartFieldSettings": {
+                  "DisplayName": "Link To Content Item",
+                  "Position": "0"
+                },
+                "ContentPickerFieldSettings": {
+                  "Required": true,
+                  "Multiple": false,
+                  "DisplayedContentTypes": []
```

Comment:
There is a new property here to display all content types by default.
Suggest we use it, as a default, otherwise, as when I tried it, there was nothing in the list!